### ISSUE TITLE
Questions related to change on OS (teredo)

### ISSUE THREAD

**Issue by Choum28** [anzz1/openspy-client#6]
Hello,

I would like to have some informations about changes the client make to the operating system during first launch

During launch it seems to create 2 regedit value that is not present by default on windows 11.

HKLM\\SOFTWARE\\Policies\\Microsoft\\Windows\\TCPIP\\v6Transition
Teredo_State  = Disabled.

and HKLM\\SOFTWARE\\WOW32Node\\Policies\\Microsoft\\Windows\\TCPIP\\v6Transition
Teredo_State  = Disabled.

then launch a cmd : netsh interface teredo set state disabled (not sure about that one).

What are the impact of such change outside openspy client usage (disabling teredo) ?

To revert change back should i just remove the regedit key and do I have to also launch a new cmd.exe command like 
netsh interface teredo set state = default ?

Thanks

**Comment by anzz1** [comment]
This simply disables the old broken IPv6-to-IPv4 tunneling implementation called Teredo which is enabled by default from WinXP up to Windows 10 release 1709 which can break P2P connectivity and cause various networking problems.

Since Windows 10 release 1803 (including all versions of Windows 11) Teredo tunneling has been disabled by default like it should have always been so this policy has no effect.

Simply put, no additional action is necessary.

**Comment by Choum28** [comment]
Thank you
